docs/resources/watch.md

```diff
@@ -396,7 +396,22 @@ Optional:
 
 ## Import
 
-Xray Watches can be imported using their names, e.g.
+Import is supported using the following syntax:
+
+```sh
+terraform import xray_watch.my-watch watch-name
+```
+
+### Import with `project_key`
+
+To import watch that is in the scope of a project, you'll need to include the project key as part of the resource ID, separated by a colon (`:`).
+
+For instance, using the following config during import:
+```terraform
+resource "xray_watch" "my-watch" {
+  name        = "watch-name"
+  project_key = "my-project"
+}
```

Comment:
Please add this to the template instead and run `make doc`

Reply:
These are the result of the running `make doc` so I don't know what you're referring to.